### PR TITLE
Update pasa to 2.5.3

### PR DESCRIPTION
**Pull request by mencian**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by joshuazhuang7** [comment]
@BiocondaBot please add label

**Review by kbseah — COMMENTED** [review]
what is the reason for removing the version ranges for transdecoder and gmap? does it prevent the environment from solving? i'm afraid it's not too clear to me.

the Dockerfile in the pasa repo does specify the versions for several dependencies, should these be used here?

**Comment by mencian** [comment]
I removed the version ranges for Transdecoder and GMAP because the [documentation](https://github.com/PASApipeline/PASApipeline/wiki/Pasa_installation_instructions) does not mention specifc or minimum versions of tools to be used. Just to be sure, I've built the recipe locally, with and without version restraints, and it seems to have no effect on the environment.

File: recipes/pasa/meta.yaml [hunk 1]
```diff
@@ -1,51 +1,43 @@
...
 test:
   commands:
-    - "${PREFIX}/opt/{{ name }}-{{ version }}/bin/cln2qual 2>&1 | grep \"cln2qual <cln_report> <qual_file>\""
-    - "${PREFIX}/opt/{{ name }}-{{ version }}/bin/mdust -h 2>&1 | grep \"Usage: mdust\""
-    - "${PREFIX}/opt/{{ name }}-{{ version }}/bin/pasa 2>&1 | grep \"Usage:.*pasa\""
-    - "${PREFIX}/opt/{{ name }}-{{ version }}/Launch_PASA_pipeline.pl --version 2>&1 | grep \"^PASA version:\""
-    - "${PREFIX}/opt/{{ name }}-{{ version }}/bin/seqclean -h 2>&1 | grep \"seqclean <seqfile>\""
-    - "${PREFIX}/opt/{{ name }}-{{ version }}/bin/trimpoly -h 2>&1 | grep \"Usage:\""
-    - "fasta 2>&1 | grep \"^ version:\""
+    - "Launch_PASA_pipeline.pl --version 2>&1"
```
Comment (bgruening) [inline recipes/pasa/meta.yaml:40]: This is only testing the perl script but not the compiled binaries, isn't it?

**Comment by kbseah** [comment]
thanks for checking. i still think that we should pin versions if they are specified in the [Dockerfile](https://github.com/PASApipeline/PASApipeline/blob/master/Docker/Dockerfile), because the developers recommend installing with Docker so it should be closest to the intended behavior.

here's the version numbers that are specified there:

```
 - gmap =2021-07-23 # not available on Bioconda, closest are 2021.08.25 or 2021.05.27, or we could build 2021-07-23
 - pblat =2.5
 - minimap2 =2.22
 - fasta3 =36.3.8h
```

File: recipes/pasa/meta.yaml (hunk 1 above)
Comment (mencian) [reply recipes/pasa/meta.yaml:40]: Correct, though I am not sure why I removed them in the first place... I have added them back in my next commit.

**Comment by mencian** [comment]
> thanks for checking. i still think that we should pin versions if they are specified in the [Dockerfile](https://github.com/PASApipeline/PASApipeline/blob/master/Docker/Dockerfile), because the developers recommend installing with Docker so it should be closest to the intended behavior.
> 
> here's the version numbers that are specified there:
> 
> ```
>  - gmap =2021-07-23 # not available on Bioconda, closest are 2021.08.25 or 2021.05.27, or we could build 2021-07-23
>  - pblat =2.5
>  - minimap2 =2.22
>  - fasta3 =36.3.8h
> ```

Thanks for providing the Dockerfile and versions! I have added minimum versions in my next commit.

**Review by kbseah — APPROVED** [review]
thanks for updating this recipe, looks good to me

**Comment by mencian** [comment]
Hello @bgruening, sorry to ping you directly, but would you be able to review the changes in this PR along with the rmblast [PR](https://github.com/bioconda/bioconda-recipes/pull/41406) when you get a chance?

**Pull request closed without merging**